# 14.2 Tidal period and the moon's revolution

Tides are caused by the gravitational pull of the moon on the earth's water, stretching it into an eliptical shape.

![exaggerated drawing of the eliptical shape of the earth's water](images/tide-eliptical.png)

So the side of the earth facing the moon, as well as the opposite side, will experience high tide. The sides perpendicular to the direction of the moon will experience a low tide.

As the earth rotates on its axis, each spot on the earth will experience two high tides every rotation.

How long will this take?

It seems reasonable to argue that there should be two high tides every 24 hours. However this is missing two important factors. The first of these is that the moon revolves around the earth (in the same direction as the earth's rotation) and so during 24 hours the moon will have moved through a portion of its revolution.

Looking at this from above the north pole, we could define and angle, $\theta$, to be the "extra" amount that the earth would need to rotate in order for the same spot on the surface to be under the moon.

![earth-moon system shown from above, with angle of rotation](images/earth-moon-system-from-above.png)

We know the rotational period of the earth is 24 hours and the orbital period of the moon, $T_M$, is 29 days, 12 hours, 44 minutes, and 2.8 seconds.

In [1]:
T_M = 29*24 + 12 + 44/60 + 2.8/(60*60)
print(T_M, 'hours for one orbit of the moon')

708.7341111111111 hours for one orbit of the moon


So $T_E = 24$ hours and $T_M = 708.734$ hours.

This means that the rotational speed of the earth is $\omega_E = \frac{1}{24}$ cycles per hour and the orbital speed of the moon is $\omega_M = \frac{1}{708.734}$ cycles per hour.

Use the equation $\omega = \frac{\theta}{t}$ to find the time that it would take for a spot on the earth to line up with the moon again.

As a hint, we know that the time for the earth to rotate to that position must be the same as the time for the moon to revolve to that position. This means we can say $t_E = t_M = T_E + \frac{\theta}{\omega_E} = \frac{\theta}{\omega_M}$. Find $t_E$ or $t_M$.

In [ ]:
T_E = 24
T_M = 29*24 + 12 + 44/60 + 2.8/(60*60)
omega_E = 1/T_E
omega_M = 1/T_M

# fill in your equation below then run this cell

t_M = 

print(t_M)